ts/screens/profile/SecurityScreen.tsx

```diff
@@ -174,37 +185,20 @@ const SecurityScreen = (): React.ReactElement => {
             <Divider />
```

Comment:
If we completely hide the **_Biometric_** switch on devices that can't handle biometrics, this `Divider` will still be visible. The last element shouldn't render the `Divider`.
I suggest moving it to the next condition (`FlatList` handles the separator automatically, but it's probably overkill here).

Reply:
@dmnplb Great suggestion! I prefer not to use a `FlatList`, but I'll follow your advice and move the `Divider` above each element that is displayed conditionally!